### PR TITLE
ADD sb item search

### PR DESCRIPTION
**Pull request by Dominik2809**
### All changes meet the following requirements
- [ ] Changelog entry was added
- [x] Changes have been tested by the author
- [x] Changes have been tested by the reviewer
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: resources/views/Category/Item/CategoryItem.twig [hunk 1]
```diff
@@ -6,7 +6,13 @@
 
 {% block PageBody %}
 
-    {% set shopBuilderTemplate = shop_builder_category_template(category.id, "categoryitem") %}
+    {% set listType = "categoryitem" %}
+    {% if isSearch %}
+        {% set listType = "itemsearch" %}
+    {% endif %}
```
Comment (Tim-M-S) [inline resources/views/Category/Item/CategoryItem.twig]: ersetzbar durch {% set listType = isSearch ? "itemsearch" : "categoryitem" %}

**Review by Tim-M-S — CHANGES_REQUESTED** [review]

File: src/ShopBuilder/Handler/ShopBuilderSettingsHandler.php [hunk 2]
```diff
@@ -123,6 +126,16 @@ public function writeRouting_MyAccountEnableRoute($enableMyAccountRoute)
     {
         return $this->setEnabledRoute(RouteConfig::MY_ACCOUNT, $enableMyAccountRoute);
     }
+    
+    public function readRouting_searchEnableRoute()
+    {
+        return in_array( RouteConfig::SEARCH, RouteConfig::getEnabledRoutes());
```
Comment (stentrop) [inline src/ShopBuilder/Handler/ShopBuilderSettingsHandler.php]:
```suggestion
        return in_array(RouteConfig::SEARCH, RouteConfig::getEnabledRoutes());
```

File: src/ShopBuilder/Handler/ShopBuilderSettingsHandler.php [hunk 3]
```diff
@@ -123,6 +126,16 @@ public function writeRouting_MyAccountEnableRoute($enableMyAccountRoute)
     {
         return $this->setEnabledRoute(RouteConfig::MY_ACCOUNT, $enableMyAccountRoute);
     }
+    
+    public function readRouting_searchEnableRoute()
```
Comment (stentrop) [inline src/ShopBuilder/Handler/ShopBuilderSettingsHandler.php]:
```suggestion
    public function readRouting_searchEnableRoute(): bool
```

File: src/ShopBuilder/Handler/ShopBuilderSettingsHandler.php [hunk 4]
```diff
@@ -123,6 +126,16 @@ public function writeRouting_MyAccountEnableRoute($enableMyAccountRoute)
...
+    {
+        return in_array( RouteConfig::SEARCH, RouteConfig::getEnabledRoutes());
+    }
+    
+    public function writeRouting_searchEnableRoute($enableSearchRoute)
```
Comment (stentrop) [inline src/ShopBuilder/Handler/ShopBuilderSettingsHandler.php]:
```suggestion
    public function writeRouting_searchEnableRoute($enableSearchRoute): string
```

**Review by stentrop — CHANGES_REQUESTED** [review]

**Review by stentrop — APPROVED** [review]

**Review by Tim-M-S — APPROVED** [review]

**Pull request merged**